.github/workflows/scripts/keep-a-changelog.bash

```diff
@@ -0,0 +1,32 @@
+
+# edit changelog for release
+#
+# Usage:
+#  cat CHANGELOG.md | bump_changelog [current version] [new version] > edited.md
+#
+function bump_changelog() {
+    CURRENT_VERSION=$1
+    NEW_VERSION=$2
+
+    while read -r line
+    do
+        if [ "$line" == "[Unreleased]: https://github.com/SpringQL/SpringQL/compare/${CURRENT_VERSION}...HEAD" ]; then 
+            # update line for [Unreleased]
+            line="[Unreleased]: https://github.com/SpringQL/SpringQL/compare/${NEW_VERSION}...HEAD"
+        fi
+
+        echo "$line"
+        if [ "$line" == "## [Unreleased]" ]; then
+            # insert line after ## [Unleleased]
+            echo # output blank line
+            echo "## [${NEW_VERSION}]"
```

Comment:
From v0.14.0, we started to add the release date here.

https://github.com/SpringQL/SpringQL/blob/main/CHANGELOG.md#v0140---2022-06-24

Could you add the date automatically from system clock? (either UTC or JST would be OK)

Reply:
Ah, sorry. How about this?

Reply:
I can do it with minor change. 
I want tests whole release workflow first.